### PR TITLE
Secrets view UX

### LINKED ISSUE #581: [UI] Secrets view UX
STR:

1. Navigate to https://taskcluster-web.netlify.com/ and login
2. Navigate to https://taskcluster-web.netlify.com/secrets/garbage%2Fhassan%2Ftest

**Expected**: The Save button is visible and shows a confirmation message when a user click on "Save".

Screen Shot 2019-04-16 at 10 29 30 AM

**Actual**: The save button is hidden inside the speed dial component and doesn't show a confirmation message when a user clicks on the save icon.

Screen Shot 2019-04-16 at 10 29 01 AM
Screen Shot 2019-04-16 at 10 27 07 AM

Inspiration: Similar to something like https://taskcluster-web.netlify.com/hooks/garbage/abc456.

### PR DESCRIPTION
**Pull request by shubhamgupta2956**
Fixes #581

### CODE REVIEW

File: ui/docs/tutorial/debug-task.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ VNC connection to a running task execution environment.
 
 There are two ways to access this mode. The easiest is to enter the task's ID
 in the [task inspector](https://tools.taskcluster.net/groups). Under
-"actions", select "create interactive task". If you have permission to create
+"actions", select "Create with SSH/VNC task". If you have permission to create
```
Comment (helfi92) [inline ui/docs/tutorial/debug-task.md]: This pull-request seems to be out of an older commit. You will want to rebase on master to remove these changes from this pr.

```diff
git pull upstream master --rebase
git push origin Secrets_view_UX_581 -f
```

**Comment by shubhamgupta2956** [comment]
I apologise for not understanding the task completely.
I have one query. In the part
            `spanProps={{
                className: classNames(
                  classes.fab,
                  classes.saveSecretSpan
                ),
              }}`
I have written classNames in different line (to keep uniformity of code like in HookForm/index.jsx). It does not shows any error there but it is showing prettier error in SecretForm/index.jsx. I understand that this is only for writing code with correct indentation but it is failing many checks. What should I do in this case? Thanks.

**Comment by helfi92** [comment]
Good question! You can fix many of the linting errors directly in the terminal. Run

```bash
yarn lint --fix
```

Hope that helps.

**Comment by shubhamgupta2956** [comment]
@helfi92 what changes do I have to make after last commit. I can't get it. Please let me know about it.

**Pull request merged**